### PR TITLE
Adding fix for ES issue.

### PR DESCRIPTION
**Pull request by udayan-y21**
Signed-off-by: Udayan Yaragattikar <udayan.yaragattikar@seagate.com>

# Backend

## Problem Statement
<pre>
Story Ref (if any): https://jts.seagate.com/browse/EOS-19785
</pre>
## Unit testing on RPM done
<pre>
Yes
</pre>
## Problem Description
<pre>
CSM component was failing at deployment deu to unsupported feature not able to communicate to ES in csm_setup config command 
</pre>
## Solution
<pre>
Added the looping logic to retry the unsupported feature operation
</pre>
## Unit Test Cases
<pre>
We have tested our patch on vm in following sequence in an automated script which runs for 100 iterations..
        Teardown CSM component -> salt '*' state.apply components.csm.teardown
	Teardown Elasticsearch -> salt '*' state.apply components.misc_pkgs.elasticsearch.teardown
	Deploy Elasticsearch component -> salt '*' state.apply components.misc_pkgs.elasticsearch
	Deploy CSM component ->salt '*' state.apply components.csm
</pre>

Deployed fulstack system with custom-ci build without failure
image

Signed-off-by:

### CODE REVIEW

**Review by soniyamoholkar — APPROVED** [review]

File: csm/conf/configure.py [hunk 1]
```diff
@@ -134,7 +142,7 @@ def _rsyslog(self):
         os.makedirs(const.RSYSLOG_DIR, exist_ok=True)
         if os.path.exists(const.RSYSLOG_DIR):
             Setup._run_cmd(f"cp -f {const.SOURCE_RSYSLOG_PATH} {const.RSYSLOG_PATH}")
-            Setup._run_cmd("systemctl restart rsyslog")
+            # Setup._run_cmd("systemctl restart rsyslog")
```
Comment (kupranay) [inline csm/conf/configure.py:145]: @udayan-y21 please remove this commented line

**Review by kupranay — APPROVED** [review]

File: csm/conf/configure.py (hunk 1 above)
Comment (udayan-y21) [reply csm/conf/configure.py:145]: Okay.

Comment (udayan-y21) [reply csm/conf/configure.py]: Changes done.

**Comment by kupranay** [comment]
run vm-deployment

**Comment by kupranay** [comment]
Look s like and intermitten failure due to infra issue hence merging it

**Review by kupranay — APPROVED** [review]

**Pull request merged**